README.md

```diff
@@ -1,11 +1,28 @@
-# A Thrift implementation for Elixir
+# A Pure Elixir Thrift Library
 
 [![Build Status](https://travis-ci.org/pinterest/elixir-thrift.svg?branch=master)](https://travis-ci.org/pinterest/elixir-thrift)
 [![Coverage Status](https://coveralls.io/repos/pinterest/elixir-thrift/badge.svg?branch=master&service=github)](https://coveralls.io/github/pinterest/elixir-thrift?branch=master)
 
-This package contains a complete implementation of [Thrift](https://thrift.apache.org/) for Elixir. It includes a Thrift IDL parser, a code generator, a binary framed client and a binary framed server.
+This package contains an implementation of [Thrift](https://thrift.apache.org/) for Elixir.
+It includes a Thrift IDL parser, a code generator, a binary framed client and a binary framed server.
 
-The serialization and deserialization code that is generated by this project is highly optimized and is between **20 and 40 times faster** than the code generated by the Apache Erlang implementation. 
+The serialization and deserialization code that is generated by this project is highly
+optimized and is between **20 and 40 times faster** than the code generated by the
+Apache Erlang implementation.
+
+#### Binary protocol benchmark
+(run `mix bench bench/binary_protocol_benchmark.exs`)
+
+benchmark name                              |iterations  |average time
+--------------------------------------------|------------|-------------
+elixir serialization (left as IOList)       |     2000   | 810.53 µs/op
+elixir deserialization                      |     1000   | 1234.69 µs/op
+elixir serialization (converted to binary)  |     1000   | 1254.23 µs/op
+erlang serialization left as IOList         |      100   | 10544.31 µs/op
+erlang serialization (converted to binary)  |      100   | 11714.74 µs/op
+erlang deserialization                      |      100   | 21671.39 µs/op
+
```

Comment:
It's also common to mention the hardware and OS that were used to produce published benchmark numbers.